### DISCUSSION TITLE
about screenshot

### DISCUSSION THREAD

**Discussion by vanshunzpt** [naverz/zepeto-studio-global#22] [General]
anyone help me to put correct ? because there no video for explain step by step 
![problem2](https://user-images.githubusercontent.com/108933918/178109748-9e109319-c03a-43df-a3f4-73313d91ae12.png)
![problem1](https://user-images.githubusercontent.com/108933918/178109751-b0785ad9-9b00-48f1-9e8c-07f89a38bdb7.png)

**Comment by jhslee** [comment]
Hello @vanshunzpt! 👋 I'm Global Tech Evangelist, John! Nice to meet you!

Can you please try a few things for me? 

1. recompiling your scripts
in the project window, type in `t:typescriptasset` and select all the resulting scripts. Click "Recompile Script"
<img width="610" alt="Screen Shot 2022-07-11 at 2 49 53 PM" src="https://user-images.githubusercontent.com/105688184/178364788-8e3748e9-aeb4-4ea9-b4a9-060124b9db12.png">

2. Hook up the relevant references in your script (If the above did not work)
<img width="1070" alt="Screen Shot 2022-07-11 at 2 52 02 PM" src="https://user-images.githubusercontent.com/105688184/178365217-809d4da7-33f8-49a6-ae25-0ebb4ff90868.png">

If both of these were unsuccessful, please let me know!

**Comment by vanshunzpt** [comment]
![asdadsadafwrwr3r](https://user-images.githubusercontent.com/108933918/178821059-7d280c71-7e02-4aae-a878-7c97ff4bf445.png)

here problem but it didnt fix

**Reply by vanshunzpt** [reply]
@jhslee i didnt solve it and it has error again

**Reply by jhslee** [reply]
Hello again @vanshunzpt!

If i understand correctly, it sounds like you're attempting to move the screenshot module from our example project into your personal project. Please try the following to get the screenshot module working in your project! 

1. Import the following package file into your project by going to Assets>ImportPackage>CustomPackage
[ZEPETO_ScreenshotModule.zip](https://github.com/naverz/zepeto-studio-global/files/9162950/ZEPETO_ScreenshotModule.zip)

<img width="585" alt="Screen Shot 2022-07-21 at 2 39 40 PM" src="https://user-images.githubusercontent.com/105688184/180319845-d7cbfe3f-cae7-495f-be17-4b18c8a5741f.png">

2.  Drag the ScreenshotModule prefab into your scene.

<img width="521" alt="Screen Shot 2022-07-21 at 2 40 37 PM" src="https://user-images.githubusercontent.com/105688184/180319993-3d00b6fb-ac2c-4685-8ad5-10eab452fe5e.png">

3. Ensure the ActiveInputHandling setting is set to "Both". Go to Edit -> Project Settings->Player->Under Other Settings under Configuration is the option Active Input Handling. Select Both. Unity will restart.

<img width="903" alt="Screen Shot 2022-07-21 at 2 42 13 PM" src="https://user-images.githubusercontent.com/105688184/180320353-9b36d314-4c7a-4f12-b29b-016ea6fe3540.png">

You should be able to see the screenshot module working in your project! 

If you're still having issues, the problem might not be the screenshot module, and there might be something else going on in your project. 

Hope this helps!

-John
Zepeto Tech Evangelist

**Reply by vanshunzpt** [reply]
![sadadadafwwer](https://user-images.githubusercontent.com/108933918/180455212-e99bebd4-6e0e-4cc3-91a4-35af8510016b.png)

@jhslee there no active input handling i use unity 2020.3 as zepeto recommend

**Reply by jhslee** [reply]
Hello @vanshunzpt  
Please open the "Other Settings" dropdown,
![180455212-e99bebd4-6e0e-4cc3-91a4-35af8510016b](https://user-images.githubusercontent.com/105688184/180489405-76a7f763-5ec2-4f60-9342-def53ce92eb7.png)

Then scroll down to configuration settings, and find the Active Input Handling option. Set that setting to "Both" 😃

**Reply by vanshunzpt** [reply]
@jhslee still error i try click it

**Comment by vanshunzpt** [comment]
![aaa111](https://user-images.githubusercontent.com/108933918/198872193-dd306e2f-638b-4d26-ba92-90af4aa64cb6.png)
![aaa222](https://user-images.githubusercontent.com/108933918/198872195-b8355f22-62ab-4da2-ba5d-126020ae7d0a.png)
![aaa333](https://user-images.githubusercontent.com/108933918/198872196-b50bec1d-7abc-414a-8c17-c479db1946e9.png)


error
